crates/holochain_sqlite/src/sql/cell/agent_activity/ts_to_seq.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  seq
+FROM
+  Action
+  JOIN DhtOp ON DhtOp.action_hash = Action.hash
+WHERE
+  DhtOp.authored_timestamp >= :until
+  AND DhtOp.type = :activity
+  AND Action.author = :author
+ORDER BY
```

Comment:
I thought I'd commented before about this but I hadn't and I seem to have missed it in MM too. I'm seeing the logic now for why this doesn't need an upper bound on the query but it is returning a potentially long list of sequences where, looking at the hash version, it only really needs to return 1.

As you've already filtered for exactly the right condition with `>=` and sorted ascending, the value you want is actually the first on in the result list right? So this could be `LIMIT 1` and you have to return less from the database?

Reply:
changed as requested